From fe6101e42c8b9d91feb5c1ae10caad7a0eba7928 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Mon, 9 Nov 2015 16:11:53 +0800
Subject: [PATCH 1065/1691] MLK-11840 video: mxsfb: change 'CONFIG_PM_RUNTIME'
 to 'CONFIG_PM'

commit fe6101e42c8b9d91feb5c1ae10caad7a0eba7928 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

The 'CONFIG_PM_RUNTIME' has been eliminated and all the users
should use 'CONFIG_PM' directly. So do this change for mxsfb.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
---
 drivers/video/fbdev/mxsfb.c | 10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

diff --git a/drivers/video/fbdev/mxsfb.c b/drivers/video/fbdev/mxsfb.c
index 0662018..d00b120 100644
--- a/drivers/video/fbdev/mxsfb.c
+++ b/drivers/video/fbdev/mxsfb.c
@@ -1538,7 +1538,7 @@ static void mxsfb_shutdown(struct platform_device *pdev)
 	}
 }
 
-#ifdef CONFIG_PM_RUNTIME
+#ifdef CONFIG_PM
 static int mxsfb_runtime_suspend(struct device *dev)
 {
 	release_bus_freq(BUS_FREQ_HIGH);
@@ -1554,12 +1554,7 @@ static int mxsfb_runtime_resume(struct device *dev)
 
 	return 0;
 }
-#else
-#define	mxsfb_runtime_suspend	NULL
-#define	mxsfb_runtime_resume	NULL
-#endif
 
-#ifdef CONFIG_PM
 static int mxsfb_suspend(struct device *pdev)
 {
 	struct fb_info *fb_info = dev_get_drvdata(pdev);
@@ -1588,6 +1583,9 @@ static int mxsfb_resume(struct device *pdev)
 	return 0;
 }
 #else
+#define	mxsfb_runtime_suspend	NULL
+#define	mxsfb_runtime_resume	NULL
+
 #define	mxsfb_suspend	NULL
 #define	mxsfb_resume	NULL
 #endif
-- 
1.9.1

